docs/UserGuide.md

```diff
@@ -150,6 +150,7 @@ Format: `newtag t/TAG1…​`
 * Tag name can only contain alphanumeric characters, apostrophes, parenthesis and whitespaces.
 * Tag name cannot be empty, or consist of only whitespaces.
 * Leading and trailing spaces are ignored. e.g `newtag t/ bride's side  t/ groom's side  ` is the same as `newtag t/bride's side t/groom's side`.
+* Extra leading parameters will be ignored. e.g. `newtag 1 t/bride's side` is the same as `newtag t/bride's side`.
```

Comment:
I suggest to change the phrasing to Extra leading characters. Similarly, deletetag command should have this phrasing as well.